packages/strapi-design-system/src/ToggleCheckbox/ToggleCheckbox.js

```diff
@@ -15,55 +15,47 @@ const Label = styled.label`
 
 const ToggleCheckboxWrapper = styled(Box)`
   height: ${getThemeSize('input')};
+  background-color: ${({ theme }) => theme.colors.neutral100};
   border: 1px solid ${({ theme, disabled }) => (disabled ? theme.colors.neutral300 : theme.colors.neutral200)};
   display: inline-flex;
   cursor: ${({ disabled }) => (disabled ? 'not-allowed' : undefined)};
   // Masks the background of each value
   overflow: hidden;
+  padding: ${({ theme }) => theme.spaces[1]};
 
   ${inputFocusStyle()}
 `;
 
-const OnBox = styled(Flex)`
-  text-transform: uppercase;
-  position: relative;
-  z-index: 2;
-`;
-
-const OffBox = styled(Flex)`
-  text-transform: uppercase;
-  border-right: 1px solid ${({ theme, disabled }) => (disabled ? theme.colors.neutral300 : theme.colors.neutral200)};
+const ValueBox = styled(Flex).attrs({
```

Comment:
IMO we need a hover state on this component. WDYT? @maevalienard

Reply:
We could add it via just showing the background of each true and false :)

Reply:
@HichamELBSI I don't think we need hover styles in this case, because that's not the way the component works. It is not possible to click or focus on the individual values (true, false) - instead you only ever toggle the invisible checkbox. I think it makes sense a11y wise to do keep it like that. A hover effect might therefore even be misleading. You can see that, if you look at the `null` case - you can't click neither true or false, but you toggle the underlying checkbox.

What do you think?